### ISSUE TITLE
polyline.antPath remove method not working

### ISSUE THREAD

**Issue by robertods** [rubenspgcavalcante/leaflet-ant-path#35]
The "**remove**" method is not working. I can not find a way to delete a polyline with ant path. I have no errors in the javascript console to help me understand what is going on. 
**Leaflet 1.0.1**

```
var myLayer= L.polyline.antPath(obj.path, 
                                        {   color: obj.strokeColor,
                                            opacity: obj.strokeOpacity,
                                            weight: obj.strokeWeight                                            
                                            ,pulseColor: obj.colorPulse,
                                            delay: obj.delay                                            
                                        }).addTo(myMap);

myLayer.pause();      // Found!
myLayer.remove();   // No found :(
```

**Comment by rubenspgcavalcante** [comment]
Hum ok, I'll check that! 👍

**Comment by rubenspgcavalcante** [comment]
Released a version patch (v0.6.3) in npm and bower :)

**Comment by robertods** [comment]
Thanks! and congratulations for so useful plugin!
